frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -332,10 +388,28 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
     detailsItem &&
     `/k8s/ns/${detailsItem.subscription.metadata.namespace}/${SubscriptionModel.plural}/${detailsItem.subscription.metadata.name}?showDelete=true`;
 
+  if (_.isEmpty(filteredItems)) {
+    return (
+      <>
```

Comment:
Don't need fragment here since you're returning a single top level component.

Reply:
Also good, will make a note for a follow.